pep-0514.txt

```diff
@@ -158,25 +185,37 @@ The Tag part of the key is intended to uniquely identify an environment within
 those provided by a single company. The key name should be alphanumeric without
 spaces and stable across installations. For example, the Python language
 version, a UUID or a partial/complete hash would be appropriate; an integer
-counter that increases for each new environment may not::
+counter that increases for each new environment may not. It is expected that
+some tools will require users to type the Tag into a command line, and that the
+Company may be optional provided the Tag is unique across all Python
+installations::
 
+    HKEY_CURRENT_USER\Software\Python\ExampleCorp\examplepy
     HKEY_CURRENT_USER\Software\Python\ExampleCorp\3.6
     HKEY_CURRENT_USER\Software\Python\ExampleCorp\6C465E66
 
```

Comment:
See above - there may be reasons to pass it around automatically. (And now twenty seconds later I can think of two large companies with internal Python app distribution systems that might be able to take advantage of this...)